### ISSUE TITLE
Why is font-weight converted to weight-text?

### ISSUE THREAD

**Issue by thebuilder** [novemberfiveco/zeplin-styled-components#3]
Is there any reason for converting the `font-weight` value to `weight-text`? It also seems to add an extra `;`. 

```
export const Txt = styled.div`
  color: #ffffff;
  line-height: 1.33;
  text-align: center;
  font-size: 21px;
  weight-text: 700;;
  height: 32px;
  width: 170px;
`; 
```

**Comment by atomworks** [comment]
I'm getting this too. Seems like an odd bug.

**Comment by nickverstocken** [comment]
Thanks for reporting! I will fix this as soon as possible!

**Comment by nickverstocken** [comment]
Fixed in latest version : [v1.1.0](https://github.com/novemberfiveco/zeplin-styled-components/releases/tag/v1.1.0)
